src/stdlib.js

```diff
@@ -11,6 +11,27 @@ export const lightscriptImports = {
   "bitwiseZeroFillRightShift": "inline",
 };
 
+export const runtimeHelpers = {
```

Comment:
This doesn't sound quite right... the transform being run here is just the one specified in `.babelrc` for the plugin, which is just `env: { node: 4 }` -- that's definitely not cross browser code.

I think having cross-browser code is a good point, but might be better to paste the helper into babel-repl, set the env to Netscape Navigator 1.0, and paste the output into a plain string.

Alternatively, an extra build step could compile the helpers with a different env? But that seems like overtooling.